sys/include/net/conn.h

```diff
@@ -15,6 +15,7 @@
  * About
  * =====
  *
+ * @code
```

Comment:
Huh? Where? As far as I can see it uses both fencing and code-block by indentation. :-/